x-pack/plugins/encrypted_saved_objects/server/plugin.ts

```diff
@@ -81,6 +82,7 @@ export class Plugin {
       registerType: (typeRegistration: EncryptedSavedObjectTypeRegistration) =>
         service.registerType(typeRegistration),
       __legacyCompat: { registerLegacyAPI: (legacyAPI: LegacyAPI) => (this.legacyAPI = legacyAPI) },
+      encryptionKeyRandomlyGenerated,
```

Comment:
What do you think about renaming `encryptionKeyRandomlyGenerated` to `usingEphemeralEncryptionKey` (or similar)? I feel like that’s more descriptive, rather than making consumers know the implementation detail that a randomly generated key is lost on restart.

Reply:
That seems reasonable to me.